webdriver/tests/bidi/script/call_function/result_ownership.py

```diff
@@ -52,9 +52,13 @@ async def test_rejected_promise(bidi_session, top_context, result_ownership, sho
                          [("root", True), ("none", False), (None, False)])
 async def test_return_value(bidi_session, top_context, await_promise, result_ownership, should_contain_handle):
     result = await bidi_session.script.call_function(
-        function_declaration="async function(){return {a:1}}",
+        function_declaration="async function(){return {a: {b:1}}}",
         await_promise=await_promise,
         result_ownership=result_ownership,
         target=ContextTarget(top_context["context"]))
 
     assert_handle(result, should_contain_handle)
+
+    # Regardless of the result_ownership, check that the nested object has no
+    # handle property.
+    assert result["value"] == [["a", {"type": "object"}]]
```

Comment:
I wonder if we should better update `assert_handle` to recursively check that the `handle` is only part of the root object?